src/proto/generation.proto

```diff
@@ -448,6 +463,7 @@ message Request {
   }
   optional ConditionerParameters conditioner = 6;
   reserved 9,10; //9, 10 are reserved
+  optional FineTuningParameters fine_tuning_parameters = 13;
```

Comment:
Should this live inside ImageParameters instead? You'd want to specify which fine tuned model to use when generating an image but when performing classification, asset get/put, interpolation, transform it would not apply correct?

Reply:
Yes that's a great point.  Thanks!